Fix JSON_OBJECT grammar in documentation

lukaseder opened the pull request:
The `JSON_OBJECT()` function allows repetitions also with the `KEY` .. `VALUE` .. syntax

lukaseder committed:
Fix JSON_OBJECT grammar in documentation

The `JSON_OBJECT()` function allows repetitions also with the `KEY` .. `VALUE` .. syntax

katzyn commented on h2/src/docsrc/help/help.csv:
```diff
@@ -5878,7 +5878,7 @@ H2VERSION()
 "Functions (JSON)","JSON_OBJECT","
 JSON_OBJECT(
-[{[KEY] string VALUE expression} | {string : expression} [,...]]
+[{[KEY] string VALUE expression} [,...] | {string : expression} [,...]]
```
Please, change it to `[{{[KEY] string VALUE expression} | {string : expression}} [,...]]` instead. Mixed syntax is perfectly valid in both SQL:2016 and H2's implementation.

lukaseder replied:
My bad. Yes, it works

lukaseder replied:
Additional commit added: https://github.com/h2database/h2database/pull/2495/commits/63d67b6163403847f064673e3ecc92cdead97b31

lukaseder committed:
Update help.csv

Additional fix as discussed here: https://github.com/h2database/h2database/pull/2495#pullrequestreview-371023400

katzyn commented:
Thanks!
